app/controllers/api/sessions_controller.rb

```diff
@@ -0,0 +1,96 @@
+class Api::SessionsController < Api::ApiController
+  respond_to :json
+
+  skip_before_action :authenticate_user, only: [:refresh]
+  before_action :require_valid_session, except: [:refresh]
+
+  def active_sessions
```

Comment:
> Because I don't even think we hang on to inactive sessions.

Good to know! We should then update the route to be: 
```ruby
get "sessions" => "api/sessions#index"
```